web/controllers/coherence/json_login.ex

```diff
@@ -0,0 +1,12 @@
+defmodule CoursePlanner.JsonLogin do
+  @moduledoc """
+  This module implements a login callback for json connection
+  """
+  use CoursePlanner.Web, :controller
```

Comment:
I think it would be better to import the 2 functions we need instead of all the things from controller here

- Plug.Conn.put_status/2
- Phoenix.Controller.render/2